pages/hackathon-2023.tsx

```diff
@@ -0,0 +1,40 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Some thoughts
* Since the `hackathon-2023` page is created, technically if this is merged into `main` users would be able to access it but I don't think that's too big of an issue since there's no sensitive information. If we need too, we could setup a redirect in Vercel but again, but probs not a big deal.
* For now, this is just a template to make future tasks easier for other devs here to jump onto as they can just some code in the child components and prevent merge conflicts.

Reply:
As long as it's not in the navigation that's perfectly fine.